From: uazo <uazo@users.noreply.github.com>
Date: Wed, 29 Oct 2025 13:01:41 +0000
Subject: Disable Sticky User Activation Across Same Origin Navigation

Do not retains the user activation status after navigating
from one page to another page of the same origin.
It is currently not possible to remove activation once it has been granted.

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 .../blink/renderer/platform/runtime_enabled_features.json5    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -5072,8 +5072,8 @@
     {
       // Preserves the sticky user activation state after a page navigates to
       // another same-origin page. https://crbug.com/433729626
-      name: "StickyUserActivationAcrossSameOriginNavigation",
-      status: "stable",
+      name: "StickyUserActivationAcrossSameOriginNavigation", status: "experimental", // disabled by default
+
     },
     {
       name: "StorageBuckets",
--
